charts/spark-operator-chart/values.yaml

```
@@ -63,6 +63,9 @@ controller:
   leaderElection:
     # -- Specifies whether to enable leader election for controller.
     enable: true
+    leaseDuration: 15
+    renewDeadline: 10
+    retryPeriod: 5
```

- For `time.Duration` flags, we would better accept duration string as input e.g. `5s`, `2m`. 
- Add some comment lines so that these values will be rendered in chart README.
- Use default values suggested by controller-runtime. For `retryPeriod`, it would be `2s`.

```suggestion
    # -- Leader election lease duration.
    leaseDuration: 15s
    # -- Leader election renew deadline.
    renewDeadline: 10s
    # -- Leader election retry period.
    retryPeriod: 2s
```